.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,41 @@
+name: Continuous Integration
+
+on:
+  push:
+  schedule:
+    - cron: "0 2 * * 1"
+
+jobs:
+  prepare:
+    runs-on: windows-latest
+    steps:
+      - name: Setup MSVC compiler
+        uses: ilammy/msvc-dev-cmd@v1
```

Comment:
I really dislike using prepackaged GitHub actions from _strangers_ when few commands can do the job anyway.
These actions:
- may be not maintained and break CI without any clear reason
- may have undesirable side effects that may hide bugs that could otherwise be uncovered by CI
- reduce reproducibility of the CI process (which version of what is actually being installed?)

Reply:
I encourage using actions, on the other hand, for the following reasons:

* The CI file should be as concise as possible, and actions have been introduced precisely to this end
* If becoming unmaintained, we can anyhow introduce our set of actions, for the reasons above
* If you use specific versions via @, you are using a release of the action, so you don't suffer from changes to master.

For now, I would stick with my approach. If we think it is the case, we can introduce our own actions and keep control of them.

Reply:
I have setup a new repo to handle your point: https://github.com/ROOT-Sim/ci-actions

It won't be immediate, but we'll have our series of actions to be shared across the various repos :smile: